README.md

```diff
@@ -20,4 +20,7 @@ fb = FaceBlurAPI(
 )
 img = fb.blur_path("<path to an image>") # Opens and runs inference on image stored in the disk
 img2 = fb.blur_np(img_arr) # Runs inference on a numpy array
+img3 = fb.blur_path("<path to an image>", blur_pct=50) # set blur_pct as an optional parameter to limit max. permissible blur size w.r.t image size
 ```
+
+## To get started, [Contact Us](mailto:sales@detecttechnologies.com)
```

Comment:
You can call the section header as `Support`. Under it, you can mention that to gain access to the system, they can mail `sales@dete...`. For support on usage of the system, they can mail `support@detecttechnologies.com`